src/app/components/top/top.component.html

```diff
@@ -18,12 +18,62 @@
   class="file-container"
   [ngClass]="{ 'dark-mode-override': darkMode }"
 >
-  <app-icon class="icon" [icon]="'icon-video-blank'" (click)="openInExplorer()"></app-icon>
-  <span
-    *ngFor="let item of fileNameArray"
-    (click)="fileWordClicked(item)"
-    class="file-word"
+  <!-- Star rating --> 
+  <div 
+    class="starRating"
   >
-    {{ item }}
-  </span>
-</div>
+    <button
```

Comment:
This section is duplicating the same code as happens in the `meta-component` so it would be best to just create a new component which will have this chunk in its template, and the `setStarRating` method in its `component.ts` file 👍